### ISSUE TITLE
NeTEx feed expired on 2024-12-14

### ISSUE THREAD

**Issue by leonardehrenfried** [noi-techpark/odh-mentor-otp#230]
Today I worked on matching up the scheduled stop point IDs when I noticed that the NeTEx feed which I download from https://rapuser:rappass@web01.sta.bz.it/netex/api/v4/downloadVersion?level=4&agencyCode=IT-ITH1 has expired a few days ago.

```
<ValidBetween>
        <FromDate>2023-12-10T00:00:00</FromDate>
        <ToDate>2024-12-14T23:59:59</ToDate>
</ValidBetween>
```

Is the URL still correct? Should the new timetable published to the same one?

This is blocking me from making progress so it would be nice if it could be fixed.

cc @rcavaliere

**Comment by rcavaliere** [comment]
@leonardehrenfried yes the URL is correct, but probably there has been some issue with the update of the new planned timetable wich entered into force last Sunday, 15.12.2024. I ask immediately to solve this issue, this shouldn't happen since the data should be automatically exported and generated on a daily basis. I will let you know.

**Comment by leonardehrenfried** [comment]
I tried again today and I'm sorry to say that it's still expired.

**Comment by leonardehrenfried** [comment]
I tried today and now the feed covers 2025.
